### PR TITLE
http-response-line/v1: add unittests

### PR DESCRIPTION
**Pull request by TheKharleeci**
Includes test for DetectHttpResponseLineTest02

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

### CODE REVIEW

**Comment by TheKharleeci** [comment]
Corresponding Suricata PR: OISF/suricata#6868

File: tests/http-response-line/writepcap.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+from scapy.all import *
+
+pkts = []
+
+load_layer("http")
+pkts += Ether(dst='ff:ff:ff:ff:ff:ff', src='00:01:02:03:04:05')/ \
+    Dot1Q(vlan=6)/ \
+    IPv6(dst='1.2.3.4', src='5.6.7.8')/TCP(sport=6666, dport=63, flags='P''A')/HTTP()/HTTPRequest(Method= 'GET',Path= '/index.html',Http_Version= 'HTTP/1.0',User_Agent='This is dummy message body',Host='www.openinfosecfoundation.org',Content_Type='text/html',
```
Comment (inashivb) [inline tests/http-response-line/writepcap.py:9]: style:
- Could you please make sure there are no spaces on either side of `=` when you're adding arguments?
- Space should be there after every `,`

**Review by inashivb — CHANGES_REQUESTED** [review]
Some style related changes requested from me. Rest, LGTM. Thanks! :)

**Comment by TheKharleeci** [comment]
Followed by: #681

**Pull request closed without merging**